babel.config.js

```diff
@@ -11,5 +11,15 @@ module.exports = {
     ],
     ['@babel/preset-react'],
   ],
-  plugins: ['@babel/plugin-transform-modules-commonjs'],
+  plugins: [
+    '@babel/plugin-transform-modules-commonjs', 
+    [
+      "module-resolver", {
+        "root": ["./src"],
+        "alias": {
+          "icons": "./lib/icons/react",
```

Comment:
nice! I think we have two options here...

We can just alias `src` as root and `lib`:

```diff
-import Flex from '../layout/Flex';
-import FDS from '../../../lib/dictionary/js/styleConstants';
-import CheckIcon from '../../../lib/icons/react/CheckIcon';
+import Flex from 'components/layout/Flex';
+import FDS from 'lib/dictionary/js/styleConstants';
+import CheckIcon from 'lib/icons/react/CheckIcon';
```

**OR**

We can drill down the `lib` aliases further (like you've already started with `icons`):

```diff
-import Flex from '../layout/Flex';
-import FDS from '../../../lib/dictionary/js/styleConstants';
-import CheckIcon from '../../../lib/icons/react/CheckIcon';
+import Flex from 'components/layout/Flex';
+import FDS from 'dictionary'; // resolves to `lib/dictionary/styleConstants`
+import CheckIcon from 'icons/CheckIcon'; // resolves to `lib/icons/react/`
```

An alias is already a little bit "magic", so I'm leaning toward going all the way and making import paths as convenient as possible. What do you think?

Reply:
hmm a lot to unpack here. To be brief, I think your former solution and reasoning makes sense, and can be done with the least amount of work and thought. I think we should go ahead with that for now 👍 thanks for the suggestion.